www/src/pages/en/usage/drizzle.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Drizzle
+description: Usage of Drizzle
+layout: ../../../layouts/docs.astro
+lang: en
+---
+
+Drizzle ORM is dialect specific, slim, performant and serverless ready by design.
```

Comment:
The wording here is a bit confusing, I suggest something like this?

Drizzle is a performant and slim ORM, that supports every major serverful and serverless runtime.

Reply:
I took that description from Drizzle docs to be honest, I think it's fine but I don't mind changing it if you think its confusing.

Reply:
Oh, my mind inserted an 'a' after 'Drizzle ORM' haha, my mind was racing, didn't see what the noun was in the context before now 🥱😂

Reply:
That's fine XD, I'll mark this as resolved than!